kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/steps/KroxyliciousSteps.java

```diff
@@ -47,16 +47,19 @@ public static String consumeMessages(String namespace, String topicName, String
     }
 
     /**
-     * Consume encrypted messages.
+     * Kroxylicious will decrypt the message so to consume an encrypted message we need to read from the underlying Kafka cluster.
      *
-     * @param namespace the namespace
+     * @param clientNamespace where to run the client job
      * @param topicName the topic name
-     * @param bootstrap the bootstrap
+     * @param kafkaClusterName the name of the kafka cluster to read from
+     * @param kafkaNamespace the namespace in which the broker is operating
      * @param numberOfMessages the number of messages
      * @param timeout the timeout
      * @return the string
      */
-    public static String consumeEncryptedMessages(String namespace, String topicName, String bootstrap, int numberOfMessages, Duration timeout) {
-        return KafkaUtils.consumeEncryptedMessageWithTestClients(namespace, topicName, bootstrap, numberOfMessages, timeout);
+    public static String consumeEncryptedMessages(String clientNamespace, String topicName, String kafkaClusterName, String kafkaNamespace, int numberOfMessages,
```

Comment:
I'm tempted re-name the method to `consumeFromUnderlyingCluster` but I'm not sold on what the right level of abstraction is. @k-wall @franvila any thoughts?

Reply:
I'm not so good giving names to methods, but you are right with that new name, because it could be reusable for any other test that wants to consume from the kafka cluster directly instead of using kroxy.

Reply:
But... we are pointing to the kafkaUtils `consumeEncryptedMessageWithTestClients` method, that is not valid for all the messages received...

Sorry for that, but reviewing it again, I'm not sure if it should be renamed to a generic name.

Reply:
haha, no thats quite alright and yes thats why I was hesitating on the rename. I think I have a refactoring that make sense. Will push shortly.